README.md

```diff
@@ -18,6 +18,27 @@ npm i --save @uniswap/analytics
 
 Make sure the environment variable `REACT_APP_STAGING` is set to `false` in production.
 
+## Development
+
+When making changes to this package, first ensure your changes are designed as needed by installing your changes from a tarball. To generate the tarball (with version `0.0.1` to indicate a test version) use the following command:
+
+```bash
+yarn tarball
+```
+
+To install in the relevant repo, copy the tarball to the root of your project's `package.json` and run the following commands:
```

Comment:
We could add something along the line "`cd`/navigate to the repo where you want to install the dependency" for clarity

Reply:
Tbh, instead of copy/pasting the tarball, it's easier to just run the `yarn add` command and give it the file path after `file:<file_path>` to avoid the extra copy pasting step during development

Reply:
Agreed, but then you have to do work from your local setup to get that file path to this dev repo. I'll add a note here on the various options though!